modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -81,13 +81,20 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 //         transport <name> {
 //             ...
 //         }
+//
+//         # handle responses
+//         handle_response [header <field>|status <status>] {
+//             ...
+//         }
 //     }
 //
 // Proxy upstream addresses should be network dial addresses such
 // as `host:port`, or a URL such as `scheme://host:port`. Scheme
 // and port may be inferred from other parts of the address/URL; if
 // either are missing, defaults to HTTP.
-func (h *Handler) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
+func (h *Handler) ParseCaddyfileReverseProxy(helper httpcaddyfile.Helper) error {
```

Comment:
```suggestion
func (h *Handler) parseCaddyfileReverseProxy(helper httpcaddyfile.Helper) error {
```

I don't think this needs to be exported (it's not used anywhere else)

Reply:
It is used in [`modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go`](https://github.com/maxatome/caddy/blob/reverse_proxy-handle_response/modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go#L286).

Reply:
Oh, right, makes sense 👍